pkg/edensdn/client.go

```diff
@@ -171,7 +171,7 @@ func (client *SdnClient) sshArgs(extra ...string) (sshArgs []string) {
 	if client.SSHKeyPath == "" {
 		log.Fatal("SDN client with undefined SSHKeyPath")
 	}
-	allArgs := fmt.Sprintf("-o ConnectTimeout=5 -o StrictHostKeyChecking=no "+
+	allArgs := fmt.Sprintf("-o IdentitiesOnly=yes -o ConnectTimeout=5 -o StrictHostKeyChecking=no "+
 		"-i %s -p %d root@localhost", client.SSHKeyPath, client.SSHPort)
```

Comment:
Is the `-i %s` parameter not preventing ssh to look for other identities?

Reply:
Nope, unfortunately. See the debug output of the ssh:

```
debug1: Will attempt key: ro@dl RSA SHA256 agent
debug1: Will attempt key: ro@dl RSA SHA256 agent
debug1: Will attempt key: ro@p RSA SHA256 agent
debug1: Will attempt key: ro@fi RSA SHA256 agent
debug1: Will attempt key: kn@mi RSA SHA256 agent
debug1: Will attempt key: sss RSA SHA256 agent
debug1: Will attempt key: /eden/sdn/vm/cert/ssh/id_rsa RSA explicit
```
the eden key is at the end. 

with the `IdentitiesOnly=yes` ssh will try only identity, skipping other keys

Reply:
OK, good to know